src/charm.py

```diff
@@ -170,6 +171,13 @@ def _pebble_layer(self) -> pebble.LayerDict:
                     "startup": "enabled",
                 },
             },
+            "checks": {
```

Comment:
no checks for filebeat?

Reply:
Not as part of this PR. The scope is limited to what's needed to secure the Wazuh API. I'm adding this check because I need to make sure the API is available in order to change the default password